openstack/workflow/v2/workflows/results.go

```diff
@@ -0,0 +1,56 @@
+package workflows
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+type commonResult struct {
+	gophercloud.Result
+}
+
+// CreateResult is the response of a Post operations. Call its Extract method to interpret it as a Workflow.
+type CreateResult struct {
+	commonResult
+}
+
+// Extract helps to get a Workflow struct from a Get function.
+func (r commonResult) Extract() (*Workflow, error) {
+	c := Workflow{}
+	err := r.ExtractInto(&c)
+	return &c, err
+}
+
+// Extract helps to get created Workflow struct from a Create function.
+func (r CreateResult) Extract() ([]Workflow, error) {
+	var s struct {
+		Workflows []Workflow `json:"workflows"`
+	}
+	err := r.ExtractInto(&s)
+	return s.Workflows, err
+}
+
+// Workflow represents a workflow execution on OpenStack mistral API.
+type Workflow struct {
```

Comment:
It appears these fields are defined here:

https://github.com/openstack/mistral/blob/632e4bda67ca1e6b579607f264f258890b0c1d8e/mistral/api/controllers/v2/resources.py#L88-L101

(for future PRs, we'll need references like this to validate the returned fields and their types)

It looks like this is missing a few fields such as tags, created_at, and updated_at?

Reply:
Added missing fields